lessons/introR/main.Rmd

```diff
@@ -294,3 +294,11 @@ library(knitr)
 knit('main.Rmd') ## or can use knitr::knit('main.Rmd')
 
 ```
+
+# Challenge: Try this out for yourself! #
+
+Make a table with the means of Agriculture, Examination, Education,
+and Infant.Mortality by Fertility as a factor that is split in the
```

Comment:
Yes, that's right. So the means of Agriculture for each category of Fertility that has been converted into a factor variable. How about:

> '..with the means of Agriculture... for each category of Fertility (hint: convert it into a factor by values >50 vs <50), and when Catholic is less than ...'